rpcserver.go

```diff
@@ -3020,6 +3035,31 @@ func (r *rpcServer) GetInfo(_ context.Context,
 	}, nil
 }
 
+// GetDebugInfo returns debug information concerning the state of the daemon
+// and its subsystems. This includes the full configuration and the latest log
+// entries from the log file.
+func (r *rpcServer) GetDebugInfo(_ context.Context,
+	_ *lnrpc.GetDebugInfoRequest) (*lnrpc.GetDebugInfoResponse, error) {
+
+	flatConfig, err := configToFlatMap(*r.cfg)
```

Comment:
See my other comment. Given that we take the `Config` and render it directly to the flat map, I think we should have a step in between here that redacts sensitive values.